doc/src/hugo/content/guides/authors/design/patterns/command-event/optional-parameters.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "3 - Optional Parameters"
+description: "Optional Version of Entity"
+date: 2022-02-25T10:50:32-07:00
+draft: "false"
+weight: 10
+---
+
+# Optional Versioning
```

Comment:
How is versioning relevant? 

Also, the single # heading will conflict with the title of the page "Optional Parameters" so start with ##, although given that, I don't see the relevance of this heading.